src/FrontDoor/FrontDoor/Cmdlets/NewFrontDoorRulesEngineActionObject.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Azure.Commands.FrontDoor.Cmdlets
 {
-    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoor" + "RulesEngineActionObject"), OutputType(typeof(PSRulesEngineAction))]
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoor" + "RulesEngineActionObject", DefaultParameterSetName = FieldsWithRegularActionParameterSet), OutputType(typeof(PSRulesEngineAction))]
```

Comment:
ParameterSet `FieldsWithRegularActionParameterSet` is not defined

Reply:
Hi @wyunchi-ms , I followed the convention to add a public field in https://github.com/Azure/azure-powershell/pull/15852/files#diff-faf23f354d9f562c6e2599390b451bb1a693513998eb34dc788143a917b575bc:
`public const string FieldsWithRegularActionParameterSet = "ByFieldsWithRegularActionParameterSet";`

Is there other places I did not take care of for this parameter set?

Reply:
I suppose this ParameterSet will be applied implicitly to all Parameters properties without  ParameterSetName explicitly defined in NewFrontDoorRulesEngineActionObject.cs

Reply:
Sorry, I mean `FieldsWithRegularActionParameterSet` is not referenced in the parameter definations.